plugins/fallbacker/timer/timer_fallbacker.go

```diff
@@ -20,13 +20,15 @@ package timer
 import (
 	"time"
 
-	"github.com/apache/skywalking-satellite/internal/pkg/event"
+	"github.com/apache/skywalking-satellite/internal/pkg/config"
+	"github.com/apache/skywalking-satellite/internal/satellite/event"
 	"github.com/apache/skywalking-satellite/plugins/forwarder/api"
 )
 
 // Fallbacker is a timer fallbacker when forward fails. `latencyFactor` is the standard retry duration,
 // and the time for each retry is expanded by 2 times until the number of retries reaches the maximum.
 type Fallbacker struct {
+	config.CommonFields
```

Comment:
why all need this struct

Reply:
the common field is inject by boot process.
for example:  a plugins want to know which pipe belong to, the commonField is for this.